lib/site_prism/waiter.rb

```diff
@@ -2,18 +2,22 @@
 
 module SitePrism
   class Waiter
-    def self.wait_until_true(wait_time_seconds = default_wait_time)
+    def self.wait_until_true(wait_time = Capybara.default_max_wait_time)
       start_time = Time.now
+
       loop do
         return true if yield
-        break unless Time.now - start_time <= wait_time_seconds
+        break if Time.now - start_time > wait_time
         sleep(0.05)
       end
-      raise SitePrism::TimeoutException, 'Timed out while waiting for block to return true'
+
+      raise SitePrism::TimeoutException, wait_time
     end
 
     def self.default_wait_time
-      Capybara.respond_to?(:default_max_wait_time) ? Capybara.default_max_wait_time : Capybara.default_wait_time
+      warn 'default_wait_time is now deprecated.'
```

Comment:
If there is a plan to remove it in a specific version, that should be specified in the message

Reply:
For now there isn't. But I didn't want to just start deleting items in the codebase as Nat asked me to go easy on making sweeping changes.

We now have a restriction in terms of gems which negates the need for this method. But I'm leaving it in for now incase anyone still references it. I'll add in a note about it being removed shortly.